example/1/game.js

```diff
@@ -6,6 +6,7 @@ const game = new Cervus.Game({
   width: window.innerWidth,
   height: window.innerHeight,
   keyboard_controlled_camera: true,
+  mouse_controlled_camera: true,
```

Comment:
I wonder if it would be easier (and leaner) to leave these two properties out of the Game class and rely on setting `keyboard_controlled` and `mouse_controlled` directly on the camera entity.